packages/fxa-settings/src/components/Settings/PageRecoveryPhoneRemove/index.test.tsx

```diff
@@ -68,4 +92,63 @@ describe('PageRecoveryPhoneRemove', () => {
 
     expect(account.removeRecoveryPhone).toBeCalled();
   });
+
+  it('navigates to settings page on cancel', async () => {
+    renderWithRouter(
+      <AppContext.Provider value={mockAppContext({ account })}>
+        <SettingsContext.Provider value={mockSettingsContext()}>
+          <PageRecoveryPhoneRemove />
+        </SettingsContext.Provider>
+      </AppContext.Provider>
+    );
+
+    const user = userEvent.setup();
+    await act(async () => {
+      await user.click(screen.getByRole('link', { name: 'Cancel' }));
+    });
+
+    expect(mockNavigate).toHaveBeenCalledWith('/settings#security', {
+      replace: true,
+    });
+  });
+
+  it('shows success alert after removing recovery phone', async () => {
+    renderWithRouter(
+      <AppContext.Provider value={mockAppContext({ account })}>
+        <SettingsContext.Provider value={mockSettingsContext()}>
+          <PageRecoveryPhoneRemove />
+        </SettingsContext.Provider>
+      </AppContext.Provider>
+    );
+
+    const user = userEvent.setup();
+    await act(async () => {
+      await user.click(
+        screen.getByRole('button', { name: 'Remove phone number' })
+      );
+    });
+
+    expect(alertBar.success).toHaveBeenCalledWith('Recovery phone removed');
```

Comment:
nit: also check that navigate was called correctly?

Reply:
navigation was checked in the 'submits' test case. 👍  Would be possible to combine the two test cases but also I like the explicitness of the current test cases so...